template/tsconfig.json

```
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
```

Should be now `es2018` instead of `esnext`

```suggestion
    "target": "es2018",
```

See why here : https://typescript.nuxtjs.org/guide/setup.html#configuration